### ISSUE TITLE
v0.51 extremely sluggish with OS X 10.8.2

### ISSUE THREAD

**Issue by indieflow** [repetier/Repetier-Host-Mac#23]
Host software is extremely sluggish if at all working whilst using latest Mac operating system on Macbook Pro Retina 15".

**Comment by repetier** [comment]
I guess retina display + anti aliasing may be the reason. You can disable anti-aliasing in the properties. That should reduce time for graphic update and I hope it helps. Have no retina display to test myself.

**Comment by indieflow** [comment]
No luck, even just tried reverting to v0.31! Retina doesn't seem to want to play with Repetier! Runs perfect on my older iMac though!

**Comment by repetier** [comment]
There is nothing special for the gui, just the simple .xib files all mac programms use. So I'm wondering if it is something on the machine like memory. Have you checked in the activity monitor if there is enough ram. As soon as swapping start everythings gets slow.

Also how slow does it get and what is slow? The gui itself of the opengl refresh (see framerate at the bottom).

**Comment by indieflow** [comment]
Well found that I can run v0.30 fine on the Retina Mac, though nothing newer!

Definitely not a memory or processor problem as hardly shows on activity monitor.

Slow as in un-usable, seems laggy. Can zoom in and out on fine on print bed display, though attempt to rotate or move and nothing happens, until you zoom again, which it then jumps to new view. Also can't choose different tabs, as in G-code, print etc. Seems as though the full program won't work even if you don't alter the view, as though not just down to the graphics!

**Comment by repetier** [comment]
So you could definitely say 0.30 runs well and 0.31 starts with the problems? I ask, because that way I could track the changes between the two versions. I would have thought 0.32 to be the first with problems on retina.

**Comment by indieflow** [comment]
Yes, v0.30 runs fine with the lag problem showing in v0.31 and newer.

**Comment by illuminarti** [comment]
I'm using 0.51, downloaded today on my 15" Retina MBP, and it seems to be working fine, with any of the anti-aliasing settings. I'm on 10.8.2 as well.

**Comment by repetier** [comment]
@illuminarti Thanks for the feedback. Was already wondering if anyone else had the same problem.

I tested retina resolution with a special debug program also with success and reviewing the code changes didn't reveal any reason for the behaviour. So maybe it is not the retina display causing the problems but something else, which is different on that laptap. 

@indieflow you could start it from a console to see if you get some error output that might help.

**Comment by SugarWill** [comment]
I'm having the same issues as @indieflow. 15" MacBook Pro Retina running 10.8.2. I can't do anything within RH 0.53 or 0.51

**Comment by repetier** [comment]
Have you tried disabling antialiasing? Also @illuminarti has it working, it is the only change in the interface between the versions.

**Comment by SugarWill** [comment]
Disabling definitely helped out, but I would still consider this unusable. Clicking different tabs like 'gcode' and 'print panel' does nothing still. Thanks for your help though!

**Comment by repetier** [comment]
I always thought sluggish means slow not not working at all.
